dcos/cosmospackage.py

```diff
@@ -20,8 +20,36 @@ class Cosmos():
     def __init__(self, cosmos_url):
         self.cosmos_url = cosmos_url
 
+    def has_capability(self, capability):
+        """Check if cluster has a capability.
+
+        :param capability: capability name
+        :type capability: string
+        :return: does the cluster has capability
+        :rtype: bool
```

Comment:
the return type is bool, so instead of raising errors, we should just return false